src/syscall/str.go

```diff
@@ -22,3 +22,13 @@ func uitoa(val uint) string {
 	buf[i] = byte(val + '0')
 	return string(buf[i:])
 }
+
+// clen returns the index of the first NULL byte in n or len(n) if n contains no NULL byte.
+func clen(n []byte) int {
```

Comment:
You could perhaps call this `strlen` (although I don't particularly care about the name as it is an implementation detail).

Reply:
I copied this from upstream so I'd prefer to keep it as is.